src/OpenSage.Game/Logic/Object/Update/ProductionUpdate.cs

```diff
@@ -156,6 +156,12 @@ private void ProduceObject(ObjectDefinition objectDefinition)
             _producedUnit = _gameObject.Parent.Add(objectDefinition, _gameObject.Owner);
             _producedUnit.Transform.Rotation = _gameObject.Transform.Rotation;
             _producedUnit.Transform.Translation = _gameObject.ToWorldspace(productionExit.GetUnitCreatePoint());
+
+            var parkingPlace = productionExit as ParkingPlaceBehaviour;
```

Comment:
Can you add a TODO comment here? Presumably this is temporary code.

Reply:
Well, i was not certain about treating ParkingPlaceBehaviour at all as IProductionExit since it has a lot of extra logic. I think we either need to extend IProductionExit or check for it's existence like above. But yeah, it's a hack for now